### PR TITLE
update graphs content for transaction summary

### PR DESCRIPTION
**Pull request by imatwawana**
Adds the User Misery and Duration Distribution graph descriptions
Removes Latency Histogram description

* description for Duration Distribution graph to be added by @DominikB2014

### CODE REVIEW

File: src/docs/product/performance/transaction-summary.mdx [hunk 1]
```diff
@@ -32,14 +32,16 @@ While you can use the search field to narrow down your events, you can also filt
 
 - **Duration Breakdown:** By graphing [P50](/product/performance/metrics/#p50-threshold), [P75](/product/performance/metrics/#p75-threshold), [P95](/product/performance/metrics/#p95-threshold), [P99](/product/performance/metrics/#p99-threshold), and P100 durations distinctly, you can utilize this display to see their transaction performance over time. Toggle the display of each segment by clicking on the legend (including Releases). Also, zoom in on specific slices to investigate spikes or possible performance regressions from a release.
 
-- **Latency Histogram:** Visualize the frequency of transactions that occur within each response time segment. This histogram view can help you understand the shape of transaction response times and quickly see outliers. Click on specific segments to drill down for more granularity.
-
 - **Duration Percentiles:** This graph shows the [average transaction duration](/product/performance/metrics/#average-transaction-duration) across distinct percentiles. For example, see how different the P50 and P99 response time durations are for a given transaction and understand the rate of increase between segments.
 
+- **Duration Distribution:** TBA
```
Comment (DominikB2014) [inline src/docs/product/performance/transaction-summary.mdx:37]:
```suggestion
- **Duration Distribution:** This graph shows the volume of transactions per median duration.
```

Comment (DominikB2014) [reply src/docs/product/performance/transaction-summary.mdx:37]: @udameli I've updated the description, let me know if it makes sense to you!

Comment (udameli) [reply src/docs/product/performance/transaction-summary.mdx:37]: looks good! ty!

**Review by udameli — APPROVED** [review]

**Review by DominikB2014 — APPROVED** [review]

**Comment by DominikB2014** [comment]
@imatwawana Added the extra description :)

File: src/docs/product/performance/transaction-summary.mdx (hunk 1 above)
Comment (imatwawana) [inline src/docs/product/performance/transaction-summary.mdx:37]:
```suggestion
- **Duration Distribution:** This graph shows the volume of transactions per median duration. This view helps  you understand the spread of the transaction durations in order to determine the likelihood of different outcomes. For example, what transaction duration is the most likely for a user.
```

File: src/docs/product/performance/transaction-summary.mdx [hunk 2]
```diff
@@ -32,14 +32,16 @@ While you can use the search field to narrow down your events, you can also filt
...
+
 - **Trends:** This graph uses a modified version of [ASAP](https://arxiv.org/pdf/1703.00983.pdf) smoothing. The Trends graph shows the unsmoothed data of the Display data, along with a smoothed version of that data.
 
 - **Web Vitals:** This graph displays the [P75 threshold](/product/performance/metrics/#p75-threshold) of [First Paint](/product/performance/web-vitals/#first-paint-fp), [First Contentful Paint](/product/performance/web-vitals/#first-contentful-paint-fcp), [Largest Contentful Paint](/product/performance/web-vitals/#largest-contentful-paint-lcp), and [First Input Delay](/product/performance/web-vitals/#first-input-delay-fid) over time. This view helps you understand how the web vitals associated with the transaction are performing over time.
 
+- **User Misery:** This graph displays the User Misery score for the transaction. User Misery is a score that represents the number of unique users who have experienced load times four times the project's configured threshold.
```
Comment (imatwawana) [inline src/docs/product/performance/transaction-summary.mdx:43]:
```suggestion
- **User Misery:** This graph displays the [User Misery](/performance/metrics/#user-misery) score for the transaction. User Misery is a score that represents the number of unique users who have experienced load times four times the project's configured threshold. 
```

**Comment by imatwawana** [comment]
I can't actually approve this PR because I'm the original author, but it looks good to me. You can go ahead and merge!

File: src/docs/product/performance/transaction-summary.mdx [hunk 3]
```diff
@@ -32,14 +32,16 @@ While you can use the search field to narrow down your events, you can also filt
...
+
 - **Trends:** This graph uses a modified version of [ASAP](https://arxiv.org/pdf/1703.00983.pdf) smoothing. The Trends graph shows the unsmoothed data of the Display data, along with a smoothed version of that data.
 
 - **Web Vitals:** This graph displays the [P75 threshold](/product/performance/metrics/#p75-threshold) of [First Paint](/product/performance/web-vitals/#first-paint-fp), [First Contentful Paint](/product/performance/web-vitals/#first-contentful-paint-fcp), [Largest Contentful Paint](/product/performance/web-vitals/#largest-contentful-paint-lcp), and [First Input Delay](/product/performance/web-vitals/#first-input-delay-fid) over time. This view helps you understand how the web vitals associated with the transaction are performing over time.
 
+- **User Misery:** This graph displays the [User Misery](/performance/metrics/#user-misery) score for the transaction. User Misery is a score that represents the number of unique users who have experienced load times four times the project's configured threshold.
```
Comment (DominikB2014) [inline src/docs/product/performance/transaction-summary.mdx:43]:
```suggestion
- **User Misery:** This graph displays the [User Misery](/product/performance/metrics/#user-misery) score for the transaction. User Misery is a score that represents the number of unique users who have experienced load times four times the project's configured threshold. 
```

**Pull request merged**